mavros/include/mavros/mavros_uas.h

```diff
@@ -310,8 +313,43 @@ class UAS {
 
 	/* -*- autopilot version -*- */
 	uint64_t get_capabilities();
+
+	/**
+	 * @brief Function to check if the flight controller has a capability
+	 *
+	 * @param capabilities can accept a multiple capability params either in enum or int from
+	 */
+	template<typename T>
```

Comment:
While these template functions are useful, I fear they may be increasing compile time, although Im not sure how to measure. The same effect could be accomplished with overloading functions but it would be more code and more runtime indirection. Let me know if these seem problematic.

Reply:
Well, here is following pro and cons:
- template may took more time, that is a problem for raspberry-pi users, if they had to compile
- overloaded functions requires more code from the human, so more room for error

From that two statements i prefer to have less code done by hand because of principle: smaller code - better code.

Reply:
Keep them, looks good to have.